tap_quickbooks/schemas/purchases.json

```diff
@@ -274,6 +145,45 @@
               "null",
               "string"
             ]
+          },
+          "LinkedTxn": {
```

Comment:
Can we move this into the shared folder and use ref here as I can see many appearances of it.

Reply:
Updated the code with a separate schema file for the `LinkedTxn` field with ref.